### DISCUSSION TITLE
&PART INITIAL_TEMPERATURE Error!

### DISCUSSION THREAD

**Discussion by Ranhill** [firemodels/fds#13366] [Q&A]
Hi,

when I include INITIAL_TEMPERATURE in 

&PART ID='foliage', DRAG_COEFFICIENT=2.8, SAMPLING_FACTOR=10, SURF_ID='foliage', QUANTITIES='PARTICLE TEMPERATURE','PARTICLE MASS', STATIC=.TRUE., COLOR='FOREST GREEN', INITIAL_TEMPERATURE=30/

it gives error in FDS 6.8.0. Can anyone please tell what is the problem?

Thanks

**Comment by mcgratta** [comment]
Does the User's Guide indicate that `INITIAL_TEMPERATURE` is a parameter that is appropriate for the `PART` line?

**Reply by Ranhill** [reply]
A sort of similar line is mentioned:
![image](https://github.com/user-attachments/assets/aa45be1e-a3b5-45cc-b664-010097773980)
Or, maybe I misunderstood it. Can you please tell me how can I change the temperature of the particle as asked above?

**Comment by drjfloyd** [comment]
That excerpt is for liquid droplets. You have a solid particle with a
SURF_ID, not a liquid droplet. You need to set the details on the SURF.

**Reply by Ranhill** [reply]
Thanks, I will check.

**Comment by Ranhill** [comment]
Hi,

I used INITIAL_TEMPERATURE with the SURF line and it gave error.

I also found two other alternatives like TMP_FRONT and TEMP_INNER. It is not clear which one should be used as only I am only trying to assign some initial temperature which will change over the course of the simulation. 

Please guide, thanks!

**Reply by drjfloyd** [reply]
A good way to become proficient with FDS is to make simple test cases to see how inputs work. In this case since you are trying to determine which of two possible inputs allow for a particle to have an initial temperature but then change temperature over time you could make a small box, put in two particles (one with each input), output the particle temperature, run FDS, and look to see which particle changes temperature over time.

**Reply by Ranhill** [reply]
Thanks, I will try that!
